CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        public IJokePrinter jokePrinter {get; private set;}
+        public IJokeService jokeService {get; private set;}
```

Comment:
Properties should be named with PascalCase
```suggestion
        public IJokePrinter JokePrinter {get; private set;}
        public IJokeService JokeService {get; private set;}
```